source/extensions/common/wasm/context.cc

```diff
@@ -1091,6 +1103,7 @@ bool Context::validateConfiguration(absl::string_view configuration, PluginShare
 }
 
 bool Context::onConfigure(absl::string_view plugin_configuration, PluginSharedPtr plugin) {
+  DeferAfterCallActions actions(this);
   if (!wasm_->on_configure_) {
     return true;
   }
```

Comment:
Those could be moved to after the check if the callback actually exist, right? Otherwise, we're wasting cycles.

(Same in most other places)

Reply:
True, but it was more obviously correct, but Done.